DigitalLearningSolutions.Web/ViewModels/LearningMenu/InitialMenuViewModel.cs

```diff
@@ -4,11 +4,17 @@
 
     public class InitialMenuViewModel
     {
-        public string Name { get; }
+        public string Title { get; }
+        public string AverageDuration { get; }
+        public string CentreName { get; }
+        public string? BannerText { get; }
 
         public InitialMenuViewModel(CourseContent courseContent)
         {
-            Name = $"{courseContent.ApplicationName} - {courseContent.CustomisationName}";
+            Title = courseContent.Title;
+            AverageDuration = courseContent.AverageDuration;
+            CentreName = courseContent.CentreName;
+            BannerText = courseContent.BannerText;

```

Comment:
You might as well just store the course content object now? 🙂

Reply:
Yeah that's much neater. Refactored in 4ec78e9 🙂